### ISSUE TITLE
Missing command env on Windows

### ISSUE THREAD

**Issue by jdegenstein** [bernhard-42/vscode-cadquery-viewer#4]
I am not sure what is the purpose of `env -u CONDA_PREFIX_1 ` in the following snippet that fails on Windows. It does not match the other commands in `vscode-cadquery-viewer/package.json`.
```
"cadquery": {
    "pip": [
        "env -u CONDA_PREFIX_1 {python} -m pip install git+https://github.com/cadquery/cadquery.git"
```

**Comment by bernhard-42** [comment]
Good catch, this is necessary to install cadquery via `pip` and works under Linux and Mac:
https://github.com/CadQuery/cadquery/issues/1236#issuecomment-1383222109 
You probably can remove that on Windows. If it then installs CadQuery 0.0.0, you need some way to unset the CONDA_PREFIX_1 variable before installation with `pip` works

**Comment by jdegenstein** [comment]
Can this command be automatically customized for each platform?

**Comment by bernhard-42** [comment]
do you know how it would look like under windows?
I can introduce another template var `{unset_env_var}` and set it accordingly on unix / windows. But would need to know how it works under windows

**Comment by bernhard-42** [comment]
@jdegenstein This should be solved in the meantime, right?
